src/main/java/website/frontrow/board/Player.java

```diff
@@ -72,4 +84,50 @@ public void addToLog(String action)
     {
         Log.add(action);
     }
+    
+    /**
+     * A getter for lives.
+     * @return lives integer
+     */
+    public int getLives() 
+    {
+    	return lives;
+    }
+    
+    /**
+     * A setter for lives.
+     * @param l integer
+     */
+    public void setLives(int l)
+    {
+    	lives = l;
+    	addToLog("The player's current amount of lives is now: " + lives);
```

Comment:
Please, use the logger format of: [CLASS_NAME]\t[Additional Note if applicable]\t<"Message">

In other words add: [PLAYER]/t to the front of the String.

<naming,organisation